src/main/resources/languages/en/messages.yml

```diff
@@ -346,7 +346,7 @@ smithing_table:
   not-working: '&4You cannot use a Slimefun item as a smithing material!'
 
 villagers:
-  no-trading: '&4You cannot trade Slimefun items with Villagers!'
+  no-trading: '&4You cannot trade this Slimefun item with Villagers or Wandering Traders!'
```

Comment:
You should define a new key since this change will neither transfer to other languages nor existing servers.